### PR TITLE
created product inquiries

### PR DESCRIPTION
**Pull request by Ksenaaa**

### CODE REVIEW

File: routes/news.routes.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ router.get(
   '/main-news',
   async (req, res) => {
     try {
-      const { dataModel } = await lastItem(newsModel, 3)
+      const { dataModel } = await lastItem(newsModel, '', '', 3)
```
Comment (vell28) [inline routes/news.routes.js]: if we have more than  3 args, we need to pass object to func args

File: routes/product.routes.js [hunk 2]
```diff
@@ -0,0 +1,108 @@
+const { Router } = require('express') 
+const router = Router()
+const productModel = require('../models/product')
+const lastItem = require('../utils/lastItem')
+
+router.get(
+  '/new',
+  async (req, res) => {
+    try {
+      const { dataModel } = await lastItem(productModel, '', '', 6)
```
Comment (vell28) [inline routes/product.routes.js]: the same as above

File: routes/product.routes.js [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+
+      let products = await dataModel.map(item => ({
```
Comment (vell28) [inline routes/product.routes.js:13]: do we need pagination here?

File: utils/createProductCode.js [hunk 4]
```diff
@@ -0,0 +1,23 @@
+const createProductCode = (async(category, type, Model) => {
+  const cat = (category.slice(0, 2) + category.slice(-1)).toUpperCase()
+  const typ = type.slice(0, 2).toUpperCase()
+
+  const lastElement = await Model.find({ category: category, type: type }).limit(1).sort({$natural:-1})
```
Comment (vell28) [inline utils/createProductCode.js]: `({ category: category, type: type })` to ({ category, type })

File: routes/product.routes.js (hunk 3 above)
Comment (Ksenaaa) [reply routes/product.routes.js:13]: no, in the func above, the limit number of elements is specified

File: utils/createProductCode.js (hunk 4 above)
Comment (Ksenaaa) [reply utils/createProductCode.js]: fixed

File: routes/news.routes.js (hunk 1 above)
Comment (Ksenaaa) [reply routes/news.routes.js]: fixed

File: routes/product.routes.js (hunk 2 above)
Comment (Ksenaaa) [reply routes/product.routes.js]: fixed

**Review by vell28 — APPROVED** [review]

**Pull request merged**